lms/djangoapps/api_manager/users/views.py

```diff
@@ -916,6 +916,39 @@ def delete(self, request, user_id, course_id):
         return Response({}, status=status.HTTP_204_NO_CONTENT)
 
 
+class UsersCoursesGradesList(SecureAPIView):
+    """
+    ### The UsersCoursesGradesList view allows clients to interact with the User's gradebook across courses
+    - URI: ```/api/users/{user_id}/courses/grades?courses={course_id1,course_id2}```
+    - GET: Returns a JSON representation of the matched set from the gradebook
+        * courses (optional): Set filtering parameter consisting of comma-separated course identifiers
+
+    ### Use Cases/Notes:
+    * Use the UsersCoursesGradesList view to interact with the User's gradebook across multiple Course enrollments
+    * Use GET to retrieve all of the Course gradebook entries for the specified User
+    """
+    def get(self, request, user_id):  # pylint: disable=unused-argument
+        """
+        GET /api/users/{user_id}/courses/grades
+        """
+        grade_complete_match_range = getattr(settings, 'GRADEBOOK_GRADE_COMPLETE_PROFORMA_MATCH_RANGE', 0.01)
+        queryset = StudentGradebook.objects.filter(user=user_id)
+        response_data = []
+        for record in queryset:
+            complete_status = False
+            if record.proforma_grade <= record.grade + grade_complete_match_range:
```

Comment:
There is a tweak missing here that we have in other places, in which if the grade is 0; proforma grade is zero, but then the complete_status becomes `true` instead of `false`.

Here's an example call from the qa server (https://qa-courses.mckinsey.edx.org/api/server/users/6657/courses/grades):
```
GET /api/server/users/6657/courses/grades HTTP/1.1
Accept-Encoding: identity
X-Edx-Api-Key: apikey
Content-Type: application/json
Connection: close
User-Agent: Python-urllib/2.7
Host: qa-courses.mckinsey.edx.org
```

Response:
```
[{
	"course_id": "DD/D101/2014",
	"complete_status": true,
	"proforma_grade": 0.0,
	"current_grade": 0.0
}, {
	"course_id": "Group/CM101/2014C",
	"complete_status": true,
	"proforma_grade": 0.0,
	"current_grade": 0.0
}, {
	"course_id": "intro/INT/2014IN",
	"complete_status": true,
	"proforma_grade": 0.0,
	"current_grade": 0.0
}, {
	"course_id": "KC/KC02/2014",
	"complete_status": true,
	"proforma_grade": 0.0,
	"current_grade": 0.0
}, {
	"course_id": "Test/MU101/2014MIT",
	"complete_status": true,
	"proforma_grade": 0.0,
	"current_grade": 0.0
}]
```